packages/monitor-v2/src/bot-oo-v3/SettleAssertions.ts

```diff
@@ -0,0 +1,43 @@
+import { getContractInstanceWithProvider, Logger, MonitoringParams, OptimisticOracleV3Ethers } from "./common";
+import { logSettleAssertion } from "./BotLogger";
+
+export async function settleAssertions(logger: typeof Logger, params: MonitoringParams): Promise<void> {
+  const oo = await getContractInstanceWithProvider<OptimisticOracleV3Ethers>("OptimisticOracleV3", params.provider);
+
+  const assertions = await oo.queryFilter(oo.filters.AssertionMade(), undefined, undefined);
+  const assertionsSettled = await oo.queryFilter(oo.filters.AssertionSettled(), undefined, undefined);
+
+  const assertionsToSettle = assertions.filter(
+    (assertion) => !assertionsSettled.some((settled) => settled.args.assertionId === assertion.args.assertionId)
+  );
+
+  const setteableAssertions = [];
```

Comment:
Is strict enabled for this package? I assumed that this wouldn't work in typescript strict, but I could be wrong.

In general, prefer enabling the strict setting in the tsconfig to get more out of the typescript compiler.